ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -183,7 +183,7 @@ tests:
         curl -s -k -L ${SDKURL}/operator-sdk-linux-x86_64.tar.gz -o opt-sdk.tar.gz
         tar -C ${HOME} -xzvf opt-sdk.tar.gz && mv ${HOME}/x86_64/operator-sdk ${HOME} && rm -fr opt-sdk.tar.gz ${HOME}/x86_64
 
-        oc image extract quay.io/hypershift/hypershift-operator:latest --file=/usr/bin/hypershift && mv hypershift "${HOME}" && chmod 755 "${HOME}"/hypershift
+        oc image extract quay.io/hypershift/hypershift-operator:4.17 --file=/usr/bin/hypershift && mv hypershift "${HOME}" && chmod 755 "${HOME}"/hypershift
```

Comment:
Is it possible to use `:latest` here ?

Reply:
no. actually it will overwrite 4.17 file. if we use latest, it cause 4.17 take latest.

after master branch is switched and there is new version, I will update mater yaml.
for example, when 4.18 become development branch and there is 4.18 hypershift cli, I will update it.

Reply:
Okay, thanks